build/azure-pipelines/win32/product-build-win32.yml

```diff
@@ -82,10 +82,28 @@ steps:
     condition: and(succeeded(), ne(variables.NODE_MODULES_RESTORED, 'true'), ne(variables['NPM_REGISTRY'], 'none'))
     displayName: Setup NPM Authentication
 
+  - powershell: |
+      mkdir -Force .build/node-gyp
```

Comment:
What was the reason for avoiding `exec` here ?

Reply:
Makes sense, but why would the following be different ? It would be good to followup on this https://github.com/microsoft/vscode/blob/2f3b279c474ffaeb7f55db8102700bcc51c70fd6/build/azure-pipelines/win32/product-build-win32.yml#L126

Maybe create a debt issue for the investigation.